src/Chapter1.hs

```diff
@@ -653,7 +668,10 @@ You need to use recursion in this task. Feel free to return to it later, if you
 aren't ready for this boss yet!
 -}
 
-firstDigit n = error "firstDigit: Not implemented!"
+firstDigit :: Int -> Int
+firstDigit n = if div m 10 == 0 then m else firstDigit(div m 10)
```

Comment:
Great Job!
In Haskell all the functions and operators are separated from its arguments with spaces 👾 
So here it would look like this:

```suggestion
firstDigit n = if div m 10 == 0 then m else firstDigit (div m 10)
```